assets/root-files/Code/php_cs

```diff
@@ -0,0 +1,52 @@
+<?php
+$finder = Symfony\CS\Finder\DefaultFinder::create()
+    ->notPath('ZendTest/Code/Generator/TestAsset')
```

Comment:
@Maks3w and @weierophinney ok, I will change into `Generator/TestAsset`. Thanks!